exec/java-exec/pom.xml

```diff
@@ -652,6 +652,12 @@
       <version>${testcontainers.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.drill.contrib</groupId>
+      <artifactId>drill-iceberg-format</artifactId>
```

Comment:
Here we have a cycle... any ideas how i could solve this without moving everything to java-exec ?